cardano-db-sync/src/Cardano/DbSync/Plugin/Default/Insert.hs

```diff
@@ -165,7 +165,7 @@ insertTx tracer blkId (tx, blockIndex) = do
     -- Insert outputs for a transaction before inputs in case the inputs for this transaction
     -- references the output (not sure this can even happen).
     lift $ zipWithM_ (insertTxOut tracer txId) [0 ..] (toList . Ledger.txOutputs $ Ledger.taTx tx)
-    mapMVExceptT (insertTxIn tracer txId) (toList . Ledger.txInputs $ Ledger.taTx tx)
+    void $ zipWithM_ (insertTxIn tracer txId) [0 ..] (toList . Ledger.txInputs $ Ledger.taTx tx)
```

Comment:
@ksaric has checked that the `void` there does not just drop any errors (which are in the `ExceptT`.

Reply:
Yes. Yes, I have.